documentation/Data-View-Filters.md

```diff
@@ -0,0 +1,385 @@
+- an additional column is added to the data table in question, with this "FilterData" (or "filter_data" in postgres) suffix, with these computed values entered into it.
+
+#### Filter data example
+
+Say we have a "product" table with a "manufacturers_list" field that stores a listBuilder response in it:
+
+```
+{
+  "list": [
+    {
+      "LBname": {
+        "value": {
+          "text": "MedPack"
+        },
+        "isValid": true,
+        "stageNumber": 1
+      },
+      "LBtasks": {
+            ...
+            },
+            "textUnselected": "Fabricação de material Activo, Fabricação de diluente, Empacotamento e rotulagem, Segundo empacotamento, Testes química e física, Testes de esterilidade, Testes microbianos, Armazenamento, Esterilização, Liberação para fornecimento",
+            "textMarkdownList": "- Fabricação de dosagem\n",
+            "selectedValuesArray": [
+                ...
+            ],
+            "textMarkdownPropertyList": "- Fabricação de material Activo: \n- Fabricação de dosagem: Fabricação de dosagem\n- Fabricação de diluente: \n- Empacotamento e rotulagem: \n- Segundo empacotamento: \n- Testes química e física: \n- Testes de esterilidade: \n- Testes microbianos: \n- Armazenamento: \n- Esterilização: \n- Liberação para fornecimento: \n",
+            "textUnselectedMarkdownList": "- Fabricação de material Activo\n- Fabricação de diluente\n- Empacotamento e rotulagem\n- Segundo empacotamento\n- Testes química e física\n- Testes de esterilidade\n- Testes microbianos\n- Armazenamento\n- Esterilização\n- Liberação para fornecimento\n"
+            },
+            ...
+        },
+        "LBaddress": {...},
+        "LBcountry": {...},
+        }
+      },
+      {
+      "LBname": {
+        "value": {
+          "text": "General Manufacturers"
+        },
+        ...
+        }
+      }
+    ],
+    "text": ...
+    },
+```
+
+We can create a column definition called `manufacturerListFilterData` and define its `filter_expression` like this:
+
+```
+{
+  "operator": "+",
+  "children": [
+    "",
+    {
+      "operator": "objectProperties",
+      "children": [
+        "manufacturersList.list.LBname.value.text"
+      ],
+      "output": "string"
+    }
+  ]
+}
+```
+
+Which would return this output based on the original response data: `MedPack, General Manufacturers`
+
+That value is (automatically) stored in a field called "manufacturer_list_filter_data" on the "product" table, which is much more suitable for filtering.
+
+Note that it is entirely possible to create *multiple* filter data columns for a single original data column if there's several data elements in it that you wish to filter by. For example, for an ingredients list, you might want a "names" and a "types" filter data column from a single response object.
+
```

Comment:
Maybe an example?

Reply:
> **For example**, for an ingredients list, you might want a "names" and a "types" filter data column from a single response object.

Is that not an example? ;) (sorry, i didn't want to actually make one since we don't currently have a use case for it -- just pointing out that it *can* be done if we need it)

Reply:
Okay, maybe I can make it a bit clearer. I've added an example of the kind of data value that you might want to make multiple filterData columns from: 3a0dd2bf

Hopefully that clarifies it. (I don't think it needs the actual column definitions spelled out, as that's already explained earlier -- you'd just to the same thing twice.)